packages/orbit-components/src/NotificationBadge/NotificationBadge.stories.tsx

```diff
@@ -8,238 +9,97 @@ import NotificationBadge from ".";
-  return <NotificationBadge type={TYPE_OPTIONS.INFO_SUBTLE}>{content}</NotificationBadge>;
-};
 
-InfoSubtle.story = {
-  parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+  args: {
+    children: "10",
+    icon: "null",
+    type: TYPE_OPTIONS.INFO,
+    ariaLabel: "additional information for screen readers",
   },
-};
-
-InfoSubtle.args = {
-  content: "10",
-};
-
-export const Success = ({ content }) => {
-  return <NotificationBadge type={TYPE_OPTIONS.SUCCESS}>{content}</NotificationBadge>;
-};
-
-Success.story = {
-  parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
-  },
-};
-
-Success.args = {
-  content: "10",
-};
-
-export const Warning = ({ content }) => {
-  return <NotificationBadge type={TYPE_OPTIONS.WARNING}>{content}</NotificationBadge>;
-};
 
-Warning.story = {
-  parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+  argTypes: {
+    icon: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
+    },
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
   },
 };
 
-Warning.args = {
-  content: "10",
-};
-
-export const Critical = ({ content }) => {
-  return <NotificationBadge type={TYPE_OPTIONS.CRITICAL}>{content}</NotificationBadge>;
-};
+export default meta;
 
-Critical.story = {
-  parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
-  },
-};
+type Story = StoryObj<typeof NotificationBadge>;
 
-Critical.args = {
-  content: "10",
-};
+export const Default: Story = {
+  render: ({ children }) => <NotificationBadge>{children}</NotificationBadge>,
 
-export const CriticalSubtle = ({ content }) => {
-  return <NotificationBadge type={TYPE_OPTIONS.CRITICAL_SUBTLE}>{content}</NotificationBadge>;
-};
-
-CriticalSubtle.story = {
   parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["icon", "type", "ariaLabel"],
+    },
   },
 };
 
-CriticalSubtle.args = {
-  content: "10",
-};
+export const Accessibility: Story = {
+  render: ({ ariaLabel, children, icon }) => {
+    const Icon = typeof icon === "string" && getIcon(icon);
 
-export const Dark = ({ content }) => {
-  return <NotificationBadge type={TYPE_OPTIONS.DARK}>{content}</NotificationBadge>;
-};
-
-Dark.story = {
-  parameters: {
-    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+    return (
+      <NotificationBadge type="info" ariaLabel={ariaLabel} icon={Icon && <Icon />}>
+        {children}
+      </NotificationBadge>
+    );
   },
 };
 
-Dark.args = {
-  content: "10",
-};
-
-export const White = ({ content }) => {
-  return (
-    <div style={{ backgroundColor: "#46515e", padding: "10px" }}>
-      <NotificationBadge type={TYPE_OPTIONS.WHITE}>{content}</NotificationBadge>
-    </div>
-  );
-};
+export const Playground: Story = {
```

Comment:
I was considering squashing the `Accessibility` and `Playground` stories as they're almost the same (Accessibility includes `ariaLabel` attribute. What's your opinion?

Reply:
I agree. Usually (or maybe even for every component), there is more controls in the Playground story but here Accessibility story has one extra control. If we would keep the Accessibility story, it would maybe make more sense to leave there only one control -  the `ariaLabel` one. But I would prefer just keeping Default, Playground and RTL ones.